layouts/partials/site-footer.html

```diff
@@ -1,89 +1,86 @@
-<div id="footerBackground">
-    <footer class=" bg-hero-footer bg-cover ">
-        <div class="pt-[350px] font-extralight text-white">
-            <div class="max-w-7xl mx-auto px-6">
-                <div class="max-[650px]:text-center sm:grid grid-cols-3">
-                    <div class="my-10 sm:my-0">
-                        {{ $data := index site.Data site.Language.Lang }}
-    
-                        {{ if $data.footer.footer.product.enable }}
-                        {{ with $data.footer.footer.product }}
-                        <h5 class="mb-3">{{ .title }}</h5>
-                        <ul>
-                            {{ range .product_items }}
-                            <li class="pb-3">
-                                <a href="{{ .link }}" target="_blank" class="font-bold">{{ .name }}</a>
-                            </li>
-                            {{ end }}
-                        </ul>
+<footer class="bg-[#192E5B]">
```

Comment:
The main change here was replacing the background classes. All of the other changes were formatting related.